readers/elasticsearch/src/main/elasticsearch/grafana/garmadon-hdfs.json

```diff
@@ -822,6 +826,165 @@
         "transform": "json",
         "type": "table"
       },
+      {
+        "aliasColors": {},
+        "bars": false,
+        "cacheTimeout": null,
+        "dashLength": 10,
+        "dashes": false,
+        "fill": 7,
+        "gridPos": {
+          "h": 6,
+          "w": 24,
+          "x": 0,
+          "y": 27
+        },
+        "id": 156,
+        "legend": {
+          "avg": false,
+          "current": false,
+          "max": false,
+          "min": false,
+          "rightSide": true,
+          "show": true,
+          "total": false,
+          "values": false
+        },
+        "lines": true,
+        "linewidth": 1,
+        "links": [],
+        "nullPointMode": "null",
+        "paceLength": 10,
+        "percentage": false,
+        "pointradius": 2,
+        "points": false,
+        "renderer": "flot",
+        "seriesOverrides": [],
+        "stack": true,
+        "steppedLine": false,
+        "targets": [
+          {
+            "alias": "standalone",
+            "bucketAggs": [
+              {
+                "field": "timestamp",
+                "id": "2",
+                "settings": {
+                  "interval": "auto",
+                  "min_doc_count": 0,
+                  "trimEdges": 0
+                },
+                "type": "date_histogram"
+              }
+            ],
+            "hide": false,
+            "metrics": [
+              {
+                "field": "select field",
+                "id": "1",
+                "meta": {},
+                "settings": {},
+                "type": "count"
+              }
+            ],
+            "query": "event_type:FS_EVENT AND uri:$uri AND dst_path:$path AND status: FAILURE AND tags: STANDALONE",
+            "refId": "A",
+            "timeField": "timestamp"
+          },
```

Comment:
New dashboard doesn't seems to work, not able to display new graph if I import it on my docker or in our grafana preprod server

Reply:
There was no datasource field, the following commit fixes that